### PR TITLE
config: enable Cockatrice hardfork

### LINKED ISSUE #3434: Enable Cockatrice in testnet/mainnet configs
https://github.com/neo-project/neo/pull/3222.

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Close #3434.

### CODE REVIEW

File: config/protocol.mainnet.neofs.yml [hunk 1]
```diff
@@ -26,6 +26,7 @@ ProtocolConfiguration:
   Hardforks:
     Aspidochelone: 3000000
     Basilisk: 4500000
+    Cockatrice: TODO
```
Comment (AnnaShaleva) [inline config/protocol.mainnet.neofs.yml:29]: @roman-khimov, we need to decide about this and testnet configs, and after it I'll update the https://github.com/neo-project/neo/blob/master/src/Neo.CLI/config.fs.mainnet.json.

File: config/protocol.testnet.neofs.yml [hunk 2]
```diff
@@ -24,6 +24,10 @@ ProtocolConfiguration:
   - morph7.t5.fs.neo.org:50333
   VerifyTransactions: true
   P2PSigExtensions: true
+  Hardforks:
+    Aspidochelone: TODO
+    Basilisk: TODO
```
Comment (roman-khimov) [inline config/protocol.testnet.neofs.yml:29]: These are enabled from 0.

File: config/protocol.testnet.neofs.yml [hunk 3]
```diff
@@ -24,6 +24,10 @@ ProtocolConfiguration:
...
+    Cockatrice: TODO
```
Comment (roman-khimov) [inline config/protocol.testnet.neofs.yml:30]: This one needs to be checked. Likely can be from 0 as well.

File: config/protocol.mainnet.neofs.yml (hunk 1 above)
Comment (roman-khimov) [reply config/protocol.mainnet.neofs.yml:29]: 5800000, ~two weeks from now.

Comment (AnnaShaleva) [reply config/protocol.mainnet.neofs.yml]: Fixed.

File: config/protocol.testnet.neofs.yml (hunk 2 above)
Comment (AnnaShaleva) [reply config/protocol.testnet.neofs.yml]: Removed.

File: config/protocol.testnet.neofs.yml (hunk 3 above)
Comment (AnnaShaleva) [reply config/protocol.testnet.neofs.yml]: We have no significant (for NeoFS) changes included into this hardfork, and states change doesn't bother us in NeoFS testnet, thus setting it to 0 as well.

Comment (roman-khimov) [reply config/protocol.testnet.neofs.yml]: We need to check compatibility anyway, but Cockatrice is not likely to be a problem, nothing significant changed for NeoFS.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**